document/document.mdk

```diff
@@ -1,6 +1,8 @@
 ﻿Title         : Static Analysis Results Interchange Format
 Heading Base  : 2
 Logo          : false
+Toc Depth     : 4
+Heading Depth : 4
```

Comment:
Because the `suppressedInSource` property section has two sub-sections, and since these are the first level 4 sections, I increment the level of headings that Madoko includes in the TOC, and also the level of headings to which Madoko assigns heading numbers.